alia-peterson/whats-cooking
# Refactor/quotes

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- changes some double quotes to single quotes to be more uniform
- adds a meta description to improve SEO



## *Instructions for reviewer:*
*Where should the reviewer start?*
- `dist/index.html` and `src/index.html`

*How should this be manually tested?*
- open page and run through all functionality looking for errors
- run lighthouse test. Should be 100

*Any background context you want to provide?*
*What are the relevant tickets?*




## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*



## Checklist:
